modules/lang-painless/src/main/java/org/elasticsearch/painless/antlr/PainlessParser.java

```diff
@@ -2834,8 +2837,21 @@ public final LambdaContext lambda() throws RecognitionException {
       }
       setState(448);
       match(ARROW);
-      setState(449);
-      block();
+      setState(451);
+      switch ( getInterpreter().adaptivePredict(_input,38,_ctx) ) {
```

Comment:
I'm finishing up regex flags so after I make a PR for that I'll have a look at braces in lambdas.